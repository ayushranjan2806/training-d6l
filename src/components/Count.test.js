import React from 'react';
import  renderer  from 'react-test-renderer';
import Count from './Count';

test('count snapashot with different count ',()=>{
    const component =renderer.create(<Count />)
    //initial rendering 

    let tree =component.toJSON();
    //
    expect(tree).toMatchSnapshot();

    tree.children[2].props.onClick();

    tree=component.toJSON();
    expect(tree).toMatchSnapshot();

    // tree.children[3].props.onClick();
    // tree=component.toJSON();
    // expect(tree).toMatchSnapshot();





  
})