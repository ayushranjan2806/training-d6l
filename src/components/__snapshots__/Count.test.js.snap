// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`count snapashot with different count  1`] = `
<div>
  <h1>
    Count:
    0
  </h1>
  <button
    onClick={[Function]}
  >
    Increment
  </button>
  <button
    onClick={[Function]}
  >
    decrement 
  </button>
</div>
`;

exports[`count snapashot with different count  2`] = `
<div>
  <h1>
    Count:
    -1
  </h1>
  <button
    onClick={[Function]}
  >
    Increment
  </button>
  <button
    onClick={[Function]}
  >
    decrement 
  </button>
</div>
`;
